config.toml

```diff
@@ -6,6 +6,19 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 tagline = "Effective observability requires high-quality telemetry"
 sub_tagline = "**OpenTelemetry** makes robust, portable telemetry a built-in feature of cloud-native software."
 font_awesome_version = "5.8.1"
+observability = """
+The term [**observability**](https://en.wikipedia.org/wiki/Observability) stems from the discipline of [control theory](https://en.wikipedia.org/wiki/Control_theory) and refers to how well a system can be understood on the basis of the **telemetry** that it produces.
+
+In software, observability typically refers to telemetry produced by **services** and is divided into three major verticals:
+
+* [**Tracing**](https://opentracing.io/docs/overview/what-is-tracing), aka **distrbuted tracing**, provides insight into the full lifecycles, aka *traces*, of requests to the system, allowing you to pinpoint failures and performance issues.  
+* [**Metrics**](https://opencensus.io/stats) provide quantitative information about processes running inside the system, including counters, gauges, and histograms. This domain also encompasses querying that data and configuring automated responses to quantitatively specified abormal states.
+* [**Logging**](https://en.wikipedia.org/wiki/Log_file) provides insight into application-specific messages emitted by processes.
+
```

Comment:
I'd add a sentence like:

Those verticals are interconnected. **Metrics** can be used to pinpoint subset of misbehaving **traces**. **Logs** associated with those traces would help to find the root cause of this behavior. And than new **metrics** can be configured based on this discovery to catch this issue earlier the next time.

Reply:
+1

Reply:
I love this